tools/cldr-code/src/main/java/org/unicode/cldr/draft/Misc.java

```diff
@@ -144,12 +144,11 @@ private static void showDefaultContent(String... strings) {
 
     private static void showSortKey() {
         String[] tests = "a ä A ぁ あ ァ ｧ ア ｱ ㋐".split(" ");
-        // TODO: freeze the Collator; problematic since changed in innermost for loop below
-        // Reference: https://unicode-org.atlassian.net/browse/CLDR-7428
         RuleBasedCollator c = (RuleBasedCollator) Collator.getInstance(ULocale.ROOT);
         c.setStrength(RuleBasedCollator.QUATERNARY);
         c.setCaseLevel(true);
         c.setHiraganaQuaternary(true);
+        // Do not freeze the collator since it is changed in the innermost "for" loop below
```

Comment:
this method is only called by main() in this file

Reply:
ok